frontend/src/App.vue

```diff
@@ -56,6 +63,64 @@
             changeDrawerValueTo(value) {
                 this.showDrawer = value;
             },
+
+            keyUpOrDown(event) {
+                let handled = false;
+                this.keymap[event.key] = event.type == "keydown";
+                if (event.ctrlKey && this.keymap["s"]) {
+                    this.saveProject(this.title, true);
+                    handled = true;
+                } else if (event.shiftKey && event.ctrlKey && this.keymap["S"]) {
```

Comment:
Nie wiem co dokładnie się dzieje, ale jak klikniesz ctrl+s to potem wystarczy że klikniesz ctrl+shift i od razu odpala się trigger od ctrl+shift+s, pewnie ta mapa nie jest czyszczona albo coś i system myśli, że s jest cały czas wciśnięte.